src/components/Scenarios/Scenarios.jsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ActionSearch from 'material-ui/svg-icons/action/search';
+import { withRouter, Link } from 'react-router-dom';
+import querystring from 'querystring';
+import FlatButton from 'material-ui/FlatButton';
+import MenuItem from 'material-ui/MenuItem';
+import { DropDownMenu } from 'material-ui/DropDownMenu';
+import ScenariosFormField from './ScenariosFormField';
+import getColumns from './ScenariosColumns';
+import { buttonStyle, formFieldStyle } from './Styles';
+import { getScenariosItemTimings, getScenariosMisc, getScenariosLaneRoles } from '../../actions/index';
+import strings from '../../lang';
+import Table from '../Table';
+import Spinner from '../Spinner';
+import Error from '../Error';
+
+const minSampleSize = row => row.games > 200;
+
+const fields = {
+  itemTimings: ['hero_id', 'item'],
+  laneRoles: ['hero_id', 'lane_role'],
+  misc: ['scenario'],
+};
+
+const menuItems = [{
+  text: strings.scenarios_item_timings,
+  value: 'itemTimings',
+},
+{
+  text: strings.heading_lane_role,
+  value: 'laneRoles',
+},
+{
+  text: strings.scenarios_misc,
+  value: 'misc',
+},
```

Comment:
swapping scenario types is putting the state into loading but doesn't seem to be firing the request.